2019-01-15 19:26:50,474 ERROR: Exception on /submit_post [GET] [in D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Stuart\Repos\blog\app\main\routes.py", line 30, in submit_post
    return render_template('submit_post.html', title="Submit Post", form=form)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\flask\templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Stuart\Repos\blog\app\templates\submit_post.html", line 26, in template
    {% endblock %}
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 199, in parse_for
    body = self.parse_statements(('name:endfor', 'name:else'))
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 199, in parse_for
    body = self.parse_statements(('name:endfor', 'name:else'))
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "D:\Users\Stuart\Anaconda3\envs\flask\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'. You probably made a nesting mistake. Jinja is expecting this tag, but currently looking for 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
2019-01-15 19:28:17,609 INFO: Stublog startup [in C:\Users\Stuart\Repos\blog\app\__init__.py:79]
2019-01-15 19:28:57,715 INFO: Stublog startup [in C:\Users\Stuart\Repos\blog\app\__init__.py:79]
2019-01-15 19:30:16,282 INFO: Stublog startup [in C:\Users\Stuart\Repos\blog\app\__init__.py:79]
2019-01-15 19:31:26,960 INFO: Stublog startup [in C:\Users\Stuart\Repos\blog\app\__init__.py:79]
2019-01-15 19:33:37,193 INFO: Stublog startup [in C:\Users\Stuart\Repos\blog\app\__init__.py:79]
2019-01-30 13:49:55,152 INFO: Stublog startup [in C:\Users\Stuart\Repos\blog\app\__init__.py:79]
2019-04-15 08:35:29,317 INFO: Stublog startup [in C:\Users\Stuart\repos\blog\app\__init__.py:79]
2019-04-15 08:36:00,475 INFO: Stublog startup [in C:\Users\Stuart\repos\blog\app\__init__.py:79]
2019-04-15 08:42:09,716 INFO: Stublog startup [in C:\Users\Stuart\repos\blog\app\__init__.py:79]
2019-04-15 08:51:16,329 INFO: Stublog startup [in C:\Users\Stuart\repos\blog\app\__init__.py:79]
2019-04-15 09:26:19,036 INFO: Stublog startup [in C:\Users\Stuart\repos\blog\app\__init__.py:79]
2019-04-15 09:31:58,628 INFO: Stublog startup [in C:\Users\Stuart\repos\blog\app\__init__.py:79]
2019-04-15 09:35:15,306 INFO: Stublog startup [in C:\Users\Stuart\repos\blog\app\__init__.py:79]
2019-04-15 09:35:52,731 INFO: Stublog startup [in C:\Users\Stuart\repos\blog\app\__init__.py:79]
